cmd/migrate-snapshots/README.md

```diff
@@ -0,0 +1,30 @@
+# migrate-snapshots
+
+_tl;dr: for DigitalOcean Kubernetes Service (DOKS) users: you can ignore the instructions and the tool below -- DigitalOcean takes care of migrating your snapshots when upgrading your clusters from 1.17 to 1.18._
+
+`migrate-snapshots` is a small tool that allows to migrate existing snapshots from the v1alpha1 to the v1beta1 API. This is helpful because no official migration path exists by the Kubernetes project.
+
+The tool is focused on DigitalOcean's Block Storage snapshot driver only: snapshot objects belonging to a different `VolumeSnapShotClass` will be ignored.
+
+`migrate-snapshots` is meant to be run against a cluster that still runs on v1alpha1 snapshots. After connecting to the cluster, it will iterate over all `VolumeSnapshotContent` and `VolumeSnapshot` objects and convert them to their respective v1beta1 structures. The converted objects are stored as YAML manifest files below a directory of choice.
+
+It is assumed that the snapshots in the DigitalOcean storage system won't be deleted even if the snapshots objects in the Kubernetes cluster are, which can be achieved by changing the `deletionPolicy` on all `VolumeSnapshotContent` objects to `Retain`. Afterwards, the snapshots can be reimported by applying the YAML manifests to the upgraded or new cluster.
+
+By default, `migrate-snapshots` reads out the `$HOME/.kube/config` file. A custom kubeconfig can be specified through the `KUBECONFIG` environment variable.
+
+## Steps
+
+Here are the proposed steps:
+
+1. Run `migrate-snapshots` without the `--directory` parameter to see which objects would be persisted.
+1. Run `migrate-snapshots -directory snapshot_objects` to store all snapshot-related objects below the `snapshot_objects` directory.
```

Comment:
nit: be consistent throughout the doc with regard to use of one vs two dash arguments.